vcs/rsync.go

```diff
@@ -0,0 +1,45 @@
+package vcs
+
+import (
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+)
+
+func init() {
+	Register(newRsync, "rsync")
+}
+
+type RsyncDriver struct{}
+
+func newRsync(b []byte) (Driver, error) {
+	return &RsyncDriver{}, nil
+}
+
+func (g *RsyncDriver) HeadRev(dir string) (string, error) {
+	return "n/a", nil
```

Comment:
Can you give me a few more hints how I could improve this PR to get it eventually merged?